src/store/inputs/reducer.js

```diff
@@ -7,6 +7,10 @@ const defaultState = {
     value: 0,
     assignedLinkIds: [],
   },
+  texture: {
+    value: null,
+    assignedLinkIds: [],
+  },
```

Comment:
One thing of note here, opening up old project files, if they are missing any attributes in the default state, they just continue on without them, need to do some sort of `state = {defaultState, ...loadedState}`

Reply:
100% this